source/tools/kubernetes/dask-helm-chart.md

```diff
@@ -50,7 +50,7 @@ This ensures that dask scheduler and worker versions match.
 `[jupyter|worker].resources` explicitly requests a GPU for each worker pod and the Jupyter pod, required by many accelerated libraries in RAPIDS.
 
 `worker.dask_worker` is the launch command for dask worker inside worker pod.
-To leverage the GPUs assigned to each Pod the [`dask_cuda_worker`](https://docs.rapids.ai/api/dask-cuda/stable/index.html) command is launched in place of the regular `dask_worker`.
+To leverage the GPUs assigned to each Pod the [`dask_cuda_worker`](https://docs.rapids.ai/api/dask-cuda/{{rapids_api_docs_version}}/index.html) command is launched in place of the regular `dask_worker`.
```

Comment:
Unfortunately if you look at the build preview this link seems to be broken.

https://rapids-deployment--403.org.readthedocs.build/en/403/tools/kubernetes/dask-helm-chart/

It takes you to https://docs.rapids.ai/api/dask-cuda/%7B%7Brapids_api_docs_version%7D%7D/index.html

Reply:
this is working now 😁 

https://rapids-deployment--403.org.readthedocs.build/en/403/tools/kubernetes/dask-helm-chart/